core/src/main/java/org/apache/accumulo/core/metadata/schema/RootTabletMetadata.java

```diff
@@ -156,10 +159,11 @@ public SortedMap<Key,Value> toKeyValues() {
    * Convert this class to a {@link TabletMetadata}
    */
   public TabletMetadata toTabletMetadata() {
-    // use a stream so we don't have to re-sort in a new TreeMap<Key,Value> structure
-    Stream<SimpleImmutableEntry<Key,Value>> entries = getKeyValues();
-    return TabletMetadata.convertRow(entries.iterator(),
-        EnumSet.allOf(TabletMetadata.ColumnType.class), false);
+    TreeMap<Key,Value> entries = new TreeMap<>();
+    getKeyValues().forEach(entry -> entries.put(entry.getKey(), entry.getValue()));
+    ManagerTabletInfo.addActions(entries, Set.of(ManagementAction.NEEDS_LOCATION_UPDATE));
```

Comment:
Is this change still needed?

Reply:
No, I don't think it's needed any longer

Reply:
Removed in 9153726